python/src/optim_module.i

```diff
@@ -56,8 +56,6 @@ OT::Ceres::Initialize();
 %include OptimizationResult.i
 %include OptimizationAlgorithmImplementation.i
 %include OptimizationAlgorithm.i
-%include NearestPointCheckerResult.i
-%include NearestPointChecker.i
```

Comment:
Why removing these two API help pages?

Reply:
these internal classes wont be exposed in Python anymore